javaagent/build.gradle.kts

```diff
@@ -34,8 +34,11 @@ val javaagentLibs by configurations.creating {
 listOf(baseJavaagentLibs, javaagentLibs).forEach {
   it.run {
     exclude("io.opentelemetry", "opentelemetry-api")
+    exclude("io.opentelemetry", "opentelemetry-api-events")
     exclude("io.opentelemetry", "opentelemetry-api-logs")
     exclude("io.opentelemetry", "opentelemetry-semconv")
+    // metrics advice API
+    exclude("io.opentelemetry", "opentelemetry-extension-incubator")
```

Comment:
I think this should also be done in the example distro and agent for testing

Reply:
I checked that, and both distro and agent-for-testing depend on the javaagent module and build on the `baseJar`/full `shadowJar` configuration; which already contains a properly built bootstrap package.

Reply:
So these don't need it?
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/448b96142a19d8b6b778810fa813ff6bde21cda8/examples/distro/agent/build.gradle#L75-L79

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/448b96142a19d8b6b778810fa813ff6bde21cda8/examples/distro/testing/agent-for-testing/build.gradle#L72-L76

Reply:
Hmm, you may be right after all. Changed 👍